bindings/python/tests/test_encoding.py

```diff
@@ -0,0 +1,36 @@
+# type: ignore
+from slimt import Encoding
+from slimt.utils import toJSON
+from collections import namedtuple
+
+
+def test_basic(service, models):
+    Pair = namedtuple("Pair", ["byte", "utf8"])
+    source = "no sé"
+    model = models[1]
+    response_byte = service.translate(
+        model, [source], html=False, encoding=Encoding.Byte
+    )[0]
+    response_utf8 = service.translate(
+        model, [source], html=False, encoding=Encoding.UTF8
+    )[0]
+    utf8: AnnotatedText = response_utf8.source
+    byte: AnnotatedText = response_byte.source
+    text = Pair(
+        byte=utf8.text.encode(),
+        utf8=utf8.text,
+    )
+    sentence_count = byte.sentence_count()
+    for sentence_idx in range(sentence_count):
+        word_count = byte.word_count(sentence_idx)
+        for word_idx in range(word_count):
+            text_range = Pair(
+                byte=byte.word_as_range(sentence_idx, word_idx),
+                utf8=utf8.word_as_range(sentence_idx, word_idx),
+            )
+            expected = text.utf8[text_range.utf8.begin : text_range.utf8.end]
+            reconstructed = text.byte[
+                text_range.byte.begin : text_range.byte.end
+            ].decode("utf-8")
```

Comment:
This checks the codepath of `Byte -> UTF8` with gold truths generated from python. The codepaths of no transfer, codepaths of `UTF8 -> Byte` remain untested, hurting coverage.

Reply:
https://github.com/jerinphilip/slimt/pull/38/commits/a56bb566c75ec5c418e018809fce928d5ad90285